src/plots/piePlot.ts

```diff
@@ -275,11 +331,54 @@ namespace Plottable.Plots {
       let sectorValueAccessor = Plot._scaledAccessor(this.sectorValue());
       let dataset = this.datasets()[0];
       let data = this._getDataToDraw().get(dataset);
-      let pie = d3.layout.pie().sort(null).value((d, i) => sectorValueAccessor(d, i, dataset))(data);
+      let pie = d3.layout.pie().sort(null).startAngle(this._startAngle).endAngle(this._endAngle)
+        .value((d, i) => sectorValueAccessor(d, i, dataset))(data);
       this._startAngles = pie.map((slice) => slice.startAngle);
       this._endAngles = pie.map((slice) => slice.endAngle);
     }
 
+    private _pieCenter(): Point {
+      let a = this._startAngle < this._endAngle ? this._startAngle: this._endAngle;
```

Comment:
I've added a general comment, above the if/elses, it's a bit hard to explain exactly whats going on on each condition without drawing. Hopefully this'll do. Let me know though. 

We could also get rid of the last if on each block but I feel like stating the condition makes it easier to understand so one can picture in their head the image of the area of the whole circle the pie should take.